lib/database_connection.rb

```diff
@@ -0,0 +1,15 @@
+require 'pg'
+
+class DatabaseConnection
+  def self.setup(dbname)
+    @connection = PG.connect(dbname: dbname)
+  end
+
+  class << self
+    attr_reader :connection
+  end
+
+  def self.query(sql)
+    @connection.exec(sql)
+  end
+end
```

Comment:
I really like how you've got the database connection in a separate file/class, good class extraction.